adapters/rubicon/rubicon.go

```diff
@@ -601,6 +602,22 @@ func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *
 	return bids, nil
 }
 
+func resolveVideoSizeId(placement openrtb.VideoPlacementType, instl int8) (sizeID int) {
+	if placement != 0 {
+		if placement == 1 {
+			return 201
+		}
+		if placement == 3 {
+			return 203
+		}
+	}
+
+	if instl == 1 {
+		return 202
+	}
+	return 0
```

Comment:
In case an `Imp` element comes in still written in the "the old" way without `instl` and a `placement` and with a valid `rubiconExt.Video.VideoSizeID`, do we want to use the latter? In other words, if `resolveVideoSizeId` returns zero, do we want to try `rubiconExt.Video.VideoSizeID`, set the size to zero, or return an error?